scss/widgets/generic/gridBase/_colors.scss

```diff
@@ -202,6 +202,9 @@ $datagrid-columnchooser-overlay-border-radius: null !default;
 $datagrid-column-separator-bg: null !default;
 $datagrid-text-stub-background-image-path: null !default;
 
+$datagrid-text-link-disabled-opacity: 0.5 !default;
+$datagrid-icon-link-disabled-opacity: 0.6 !default;
```

Comment:
Did designer check these values?

Reply:
Yes, @DX-man told me to use same style as regular buttons have, so I copied them

Reply:
Instead of coping, you can use these contants directly:

```
$datagrid-text-link-disabled-opacity: $base-disabled-opacity;
$datagrid-icon-link-disabled-opacity: $button-disabled-icon-opacity;
```